src/molecules/Select/Select.tsx

```diff
@@ -55,6 +55,7 @@ export const Select = <T extends SelectOptionRequired>(
     placeholder = 'Select...',
     label,
     helperText,
+    showHelperIcon = true,
```

Comment:
Should this be on a JSDoc thing in the SingleSelect/ComboBox to say that it defaults to true?

Reply:
Sure! I'll add it